oauth2_http/java/com/google/auth/oauth2/UserAuthorizer.java

```diff
@@ -56,6 +57,17 @@
 /** Handles an interactive 3-Legged-OAuth2 (3LO) user consent authorization. */
 public class UserAuthorizer {
 
+  /**
+   * Represents the client authentication types as specified in RFC 7591.
+   *
+   * <p>For more details, see <a href="https://tools.ietf.org/html/rfc7591">RFC 7591</a>.
```

Comment:
nit: Missing closing <p>

Reply:
That's not a missing. Like the last javadoc change. it won't pass lint if I have that closing `</p>`. And the auto format tool running is to just remove it.